src/index.html

```diff
@@ -16,17 +19,34 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+  <body class="page__body">
```

Comment:
There should be no blank line between the head and body tags. Remove the empty line to maintain proper HTML formatting.